server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/litecoin/LitecoinAddressValidator.java

```diff
@@ -26,7 +26,7 @@
 
     @Override
     public boolean isAddressValid(String address) {
-        if (address.startsWith("L") || address.startsWith("3")) {
+        if (address.startsWith("L") || address.startsWith("M")) {
```

Comment:
Don't deprecate 3-address. Leave the support present and we can merge right away